xiblint/rules/no_external_storyboard_links.py

```diff
@@ -0,0 +1,10 @@
+"""
+Ensures there are no links to storyboards in different bundles
+"""
+
+
+def check(context):  # type: (xiblint.xibcontext.XibContext) -> None
+    for element in context.tree.findall(".//viewControllerPlaceholder"):
+        if element.get("bundleIdentifier"):
+            context.error(element, "Linking to external storyboard")
```

Comment:
I think you should change this to "Linking to a view controller in another bundle" which is a bit more clear